scripts/memrem.coffee

```diff
@@ -10,30 +10,55 @@
 # Commands:
 #   hubot mem <keyword> - prints out saved resource
 #   hubot get <keyword> - prints out saved resource
+#   hubot get list - prints out saved keys
 #   hubot rem <keyword> <resource> - saves resource to specified keyword
 #   hubot set <keyword> <resource> - saves resource to specified keyword
 #
 # Author:
 #   snibbles
 
-memremId = (keyword) -> "memrem_#{keyword}"
+memremBrain = "memrem"
 
 onGet = (msg) ->
-  keyword = msg.match[2]
-  key = memremId keyword
-  value = msg.robot.brain.get key
-  if value
+  key = msg.match[2]
+
+  if key is "list"
```

Comment:
Here is the current set regex:

```
/(mem|get) (?!safesearch)([a-zA-Z0-9_-]*)$/i
```

Could we just ignore list in the regex and remove the `if key is "list"` check in both set and get commands by adding `list` to the regex, like so?

```
/(mem|get) (?!safesearch|list)([a-zA-Z0-9_-]*)$/i
```